server/service/testing_client.go

```diff
@@ -148,7 +148,16 @@ func (ts *withServer) commonTearDownTest(t *testing.T) {
 		}
 
 		_, err = q.ExecContext(ctx, "DELETE FROM in_house_apps;")
-		return err
+		if err != nil {
+			return err
+		}
+
+		_, err = q.ExecContext(ctx, "DELETE FROM vpp_apps;")
```

Comment:
Thinking this should be deleting from `vpp_apps_teams` where global_or_team_id = 0? Or is the vpp_apps deletion intentional here to do broader cleanup?